docs/modules/operation/pages/events/sources/snmp-traps.adoc

```diff
@@ -240,7 +240,7 @@ As an example, the SNMP agent returns a numerical value for the `ifAdminStatus`
 Because {page-component-title} does not have a MIB parser, we usually put this map (between the numerical encoded value and its meaning) into the event description.
 Use the `varbindsdecode` element to convert the event varbind value into a decoded String.
 
-With a properly configured event, you can decode the numerical value sent into trap varbinds and the corresponding string value into the `logmsg`.
+With a properly configured event, you can decode the numerical value sent into trap varbinds and the corresponding String value into the `logmsg`.
```

Comment:
How about this?

With varbindsdecode, OpenNMS decodes the numerical value sent into trap varbinds to the corresponding String values which then can be substituted into the logmsg.